app/presenters/publishing_api/historical_account_presenter.rb

```diff
@@ -0,0 +1,53 @@
+module PublishingApi
+  class HistoricalAccountPresenter
+    attr_accessor :historical_account, :update_type
+
+    def initialize(historical_account, update_type: nil)
+      self.historical_account = historical_account
+      self.update_type = update_type || "major"
+    end
+
+    delegate :content_id, to: :historical_account
+
+    def content
+      content = BaseItemPresenter.new(
+        historical_account,
+        title: historical_account.person.name, # TODO
+        update_type:,
+      ).base_attributes
+
+      content.merge!(
+        description: historical_account.summary,
+        details:,
+        document_type: "historic_appointment",
+        public_updated_at: historical_account.updated_at,
+        rendering_app: Whitehall::RenderingApp::WHITEHALL_FRONTEND,
+        schema_name: "historic_appointment",
+      )
+
+      content.merge!(PayloadBuilder::PolymorphicPath.for(historical_account))
+    end
+
+    def links
+      {
+        person: [
+          historical_account.person.content_id,
+        ],
+      }
+    end
+
+  private
+
+    def details
+      {}.tap do |details|
```

Comment:
Is there any benefit in doing this over just writing the hash out?

Reply:
I'd just copied it from another presenter.  Looks like there's no benefit at all, so I'll change this to just write out the hash.